### ISSUE TITLE
libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: CLevelDB(): error opening database environment Corruption: missing start of fragmented record(2)

### ISSUE THREAD

**Issue by Kangou** [miguelfreitas/twister-core#337]
Twister was downloading the block chain after a long pause of some months (after a git pull and a full rebuild of course), and I had to force reboot my Mac because of some game crash, not related with Twister.

Now, when I Try to launch twisted, I get this error:
```
libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: CLevelDB(): error opening database environment Corruption: missing start of fragmented record(2)
```
Even with an empty .twister directory, I get the same error message. Is there another directory I need to clean, or do I have some corrupted system libraries ?

The best I have after a reboot and a -reindex:
```
dev/twister-core git:(master) ▶ ./twisterd
: mapBlockIndex detected but nBestHeight still zero, trying to repair (reindex).

Do you want to rebuild the block database now? (assuming YES)
libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: CLevelDB(): error opening database environment Corruption: missing start of fragmented record(2)
[1]    1296 abort      ./twisterd
```

**Comment by miguelfreitas** [comment]
no idea. looks like a common problem to bitcoin
https://github.com/bitcoin/bitcoin/issues/2305
though in their case apparently a -reindex fixed it.

**Comment by Kangou** [comment]
Yeah I've see that. I can understand that my database was corrupted after an hard reboot, but I don't understand why i'm getting this from a fresh start. I have rebooted, git clean -dxf && git checkout -f, rebuilt everything, erased the .twisterd folder and I get the same error.

**Comment by Kangou** [comment]
Okay, Fixed it ! The real blockchain was in ~/Library/Application Support/Twister, and not ~/.twisterd.
